Events/2022-03-diverse-leung.md

```diff
@@ -0,0 +1,45 @@
+# DiveRSE Speaker Series: Mary Leung
+
+- Date: 2022-03-22
+- Location: Online
+- Event Website: https://diverse-rse.github.io/events/2022-03-22
+- Organizers: DiveRSE & Society of Research Software Engineering 
+			   
+[DiveRSE (Diverse RSE)](https://diverse-rse.github.io/) is a series of talks and related activities including
+discussion and panel sessions designed to support and raise
+awareness of EDI challenges and successes within the research
+software community. The series runs from March 2022 and is linked to
+the [“Vive la différence - research software engineers” workshop](https://www.researchsoft.org/events/2022-04/)
+being hosted at the Lorentz Centre in The Netherlands in April 2022.
+
+
+
+Resource Information | Details
+:--- | :---			   
+Webinar Title | Normalizing inclusion by embracing difference
+Date and Time | 2022-03-22 16:00 UTC
+Presenter | Mary Ann Leung (Sustainable Horizons Institute)
+Registration, Information, and Archives | 	<https://diverse-rse.github.io/events/2022-03-22>	   
+
+**This 
```

Comment:
@mrmundt did you intend some content here?  Or is this just a stray that we can delete?